Sources_v3/StreamChatUI/Utils/NSLayoutConstraint+Extensions.swift

```diff
@@ -1,19 +1,14 @@
 //
-// Copyright © 2020 Stream.io Inc. All rights reserved.
+// Copyright © 2021 Stream.io Inc. All rights reserved.
 //
 
 import UIKit
 
 extension UILayoutPriority {
-    /// The `UILayoutPriority` that is less than required by 1.
-    ///
-    /// Such helper is especially handy when working with `UIStackView`:
-    /// when the arranged subview is hidden, the constraint for `height == 0`
-    /// with `.required` priority is activated.
-    ///
-    /// Having all height constraints inside the `UIStackView` with `almostRequired`
-    /// priority eliminates breaking constraints warnings in console.
-    static let almostRequired: Self = .required - 1
+    /// Having our default priority lower than `.required(1000)` allow user easily
+    /// override any default constraints and customize layout
+    static let streamRequire = UILayoutPriority(rawValue: 900)
+    static let streamAlmostRequire: UILayoutPriority = .streamRequire - 1
```

Comment:
I'd add documentation also to this variable so people do not need to go to the code and see the concrete value (it's 899)  

Example:

```
/// Having our default priority lower than `.required(1000)` allow user easily
/// override any default constraints and customize layout
/// That's why this value equals `900` instead of `1000`.
static let streamRequire = UILayoutPriority(rawValue: 900) 

/// In our space, this is `streamRequire` - `1`, i.e. 899
static let streamAlmostRequire: UILayoutPriority = .streamRequire - 1
``` 

Or something like that to make it more clear :)

Reply:
IMHO final value is implementation detail and do not bring value to documentation.
If you need their concrete values for some reason, probably something wrong.